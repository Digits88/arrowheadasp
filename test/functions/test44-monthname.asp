<!-- Test MonthName function.
-->

MonthName(1) = <%= MonthName(1) %><br>
MonthName(2) = <%= MonthName(2) %><br>
MonthName(3) = <%= MonthName(3) %><br>
MonthName(4) = <%= MonthName(4) %><br>
MonthName(5) = <%= MonthName(5) %><br>
MonthName(6) = <%= MonthName(6) %><br>
MonthName(7) = <%= MonthName(7) %><br>
MonthName(8) = <%= MonthName(8) %><br>
MonthName(9) = <%= MonthName(9) %><br>
MonthName(10) = <%= MonthName(10) %><br>
MonthName(11) = <%= MonthName(11) %><br>
MonthName(12) = <%= MonthName(12) %><br>

MonthName(1, True) = <%= MonthName(1, True) %><br>
MonthName(2, True) = <%= MonthName(2, True) %><br>
MonthName(3, True) = <%= MonthName(3, True) %><br>
MonthName(4, True) = <%= MonthName(4, True) %><br>
MonthName(5, True) = <%= MonthName(5, True) %><br>
MonthName(6, True) = <%= MonthName(6, True) %><br>
MonthName(7, True) = <%= MonthName(7, True) %><br>
MonthName(8, True) = <%= MonthName(8, True) %><br>
MonthName(9, True) = <%= MonthName(9, True) %><br>
MonthName(10, True) = <%= MonthName(10, True) %><br>
MonthName(11, True) = <%= MonthName(11, True) %><br>
MonthName(12, True) = <%= MonthName(12, True) %><br>

MonthName(1, False) = <%= MonthName(1, False) %><br>
MonthName(2, False) = <%= MonthName(2, False) %><br>
MonthName(3, False) = <%= MonthName(3, False) %><br>
MonthName(4, False) = <%= MonthName(4, False) %><br>
MonthName(5, False) = <%= MonthName(5, False) %><br>
MonthName(6, False) = <%= MonthName(6, False) %><br>
MonthName(7, False) = <%= MonthName(7, False) %><br>
MonthName(8, False) = <%= MonthName(8, False) %><br>
MonthName(9, False) = <%= MonthName(9, False) %><br>
MonthName(10, False) = <%= MonthName(10, False) %><br>
MonthName(11, False) = <%= MonthName(11, False) %><br>
MonthName(12, False) = <%= MonthName(12, False) %><br>
