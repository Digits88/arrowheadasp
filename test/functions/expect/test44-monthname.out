<!-- Test MonthName function.
-->

MonthName(1) = January<br>
MonthName(2) = February<br>
MonthName(3) = March<br>
MonthName(4) = April<br>
MonthName(5) = May<br>
MonthName(6) = June<br>
MonthName(7) = July<br>
MonthName(8) = August<br>
MonthName(9) = September<br>
MonthName(10) = October<br>
MonthName(11) = November<br>
MonthName(12) = December<br>

MonthName(1, True) = Jan<br>
MonthName(2, True) = Feb<br>
MonthName(3, True) = Mar<br>
MonthName(4, True) = Apr<br>
MonthName(5, True) = May<br>
MonthName(6, True) = Jun<br>
MonthName(7, True) = Jul<br>
MonthName(8, True) = Aug<br>
MonthName(9, True) = Sep<br>
MonthName(10, True) = Oct<br>
MonthName(11, True) = Nov<br>
MonthName(12, True) = Dec<br>

MonthName(1, False) = January<br>
MonthName(2, False) = February<br>
MonthName(3, False) = March<br>
MonthName(4, False) = April<br>
MonthName(5, False) = May<br>
MonthName(6, False) = June<br>
MonthName(7, False) = July<br>
MonthName(8, False) = August<br>
MonthName(9, False) = September<br>
MonthName(10, False) = October<br>
MonthName(11, False) = November<br>
MonthName(12, False) = December<br>
